src/components/Button/index.tsx

```diff
@@ -256,28 +241,49 @@ const ButtonErrorStyle = styled(Base)`
     background-color: ${({ theme }) => theme.red1};
   }
 `
-
-export function ButtonConfirmed({
-  confirmed,
-  altDisabledStyle,
-  ...rest
-}: { confirmed?: boolean; altDisabledStyle?: boolean } & ButtonProps) {
+export interface ButtonConfirmedProps extends Omit<ButtonProps, 'content'> {
+  confirmed?: boolean
+  altDisabledStyle?: boolean
+  content: string | JSX.Element
```

Comment:
What's the rational for this? `children` is of type `React.ReactNode` and `ReactNode`:

```typescript
type ReactText = string | number;
type ReactChild = ReactElement | ReactText;

interface ReactNodeArray extends Array<ReactNode> {}
type ReactFragment = {} | ReactNodeArray;

type ReactNode = ReactChild | ReactFragment | ReactPortal | boolean | null | undefined;
```